ui/components/MesheryPatterns.js

```diff
@@ -384,14 +389,17 @@ function MesheryPatterns({
       );
     }
 
-    if (type === FILE_OPS.FILE_UPLOAD || type=== FILE_OPS.URL_UPLOAD) {
+    if (type === FILE_OPS.FILE_UPLOAD || type=== FILE_OPS.URL_UPLOAD || type === FILE_OPS.GH_IMPORT) {
       let body;
       if (type === FILE_OPS.FILE_UPLOAD) {
         body = JSON.stringify({  pattern_data : { pattern_file : data }, save : true })
       }
       if (type === FILE_OPS.URL_UPLOAD) {
         body = JSON.stringify({ url : data, save : true })
       }
+      if (type === FILE_OPS.GH_IMPORT) {
+        body = JSON.stringify({ url : data, })
```

Comment:
As we are passing the object : 
```{ "url": finalURL, “path”: `${path}/${ check ? '**' : ''}```, “save”: false ` as parameter for the onSubmit function before, we need to construct body like this here as far as I know. You can set 
`body=data` and continue.